opal/corelib/range.rb

```diff
@@ -85,16 +93,26 @@ def exclude_end?
     @exclude
   end
 
-  alias :first :begin
+  def first(n=(n_specified=true; nil))
```

Comment:
Probably it can be improved by passing default value = `undefined`:
``` ruby
def m(n = undefined)
  %x{
    if (n == null) {
      console.log('not passed');
    } else if (n === nil) {
      console.log('nil passed');
    } else {
      console.log('value passed');
    }
  }
end

m
m(nil)
m(123)
```

Out:
```
not passed
nil passed
value passed
```